server/odc-common/src/main/java/com/oceanbase/odc/common/security/SensitiveDataUtils.java

```diff
@@ -15,18 +15,77 @@
  */
 package com.oceanbase.odc.common.security;
 
+import java.util.Iterator;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+import org.json.JSONObject;
```

Comment:
is it defined in fastjson ?

Reply:
Not fastjson, use JsonUtils instead for uniformity. In interest I also looked for the author, from [maven central repository](https://central.sonatype.com/artifact/org.json/json) know that the known contributor is Douglas Crockford, known as the father of json.